c10/metal/special_math.h

```diff
@@ -1716,5 +1716,45 @@ float chebyshev_polynomial_w_forward(T x, int64_t n) {
   return r;
 } // chebyshev_polynomial_w_forward(T x, int64_t n)
 
+template <typename T>
+unsigned short getHermitianLimit() {
+  if (sizeof(T) <= sizeof(float)) {
```

Comment:
```suggestion 
  if constexpr (sizeof(T) <= sizeof(float)) {
```

Reply:
You can't have `if constexpr` in Metal, because it's still C++14 compliant